tests/test_geome_permit_parsing.py

```diff
@@ -0,0 +1,88 @@
+      "compliesWith": []}),
+    ("research visa no. 10350008304; Dept Env & Conservation/Nat Research Institute",
+     {"authorizedBy": ["research visa no. 10350008304", "Dept Env & Conservation/Nat Research Institute"],
+      "compliesWith": []}),
+    (
+        '"Ministufffdrio da Aquicultura e Pescas, Direcufffdufffdo Nacional de Pescas e Aquicultura (authorised by A '
+        'Fernandes, L Fontes, J Freitas; guia de marssa: 502/DNPA/VIII/10 and 452/DNPA/VII/11). Export of samples was '
+        'authorised by the Departmento de Quarentena das Pescas (export permit: 162/FQ006/EXP./DNQB/VII/2011)"',
+        {"authorizedBy": [
+            "Ministufffdrio da Aquicultura e Pescas, Direcufffdufffdo Nacional de Pescas e Aquicultura (authorised by "
+            "A Fernandes, L Fontes, J Freitas; guia de marssa: 502/DNPA/VIII/10 and 452/DNPA/VII/11). Export of "
+            "samples was authorised by the Departmento de Quarentena das Pescas (export permit: "
+            "162/FQ006/EXP./DNQB/VII/2011)"],
+            "compliesWith": []}),
+    ("NMFS permits, state permits, & CITES permits",
+     {"authorizedBy": ["NMFS permits", "state permits", "CITES permits"], "compliesWith": []}),
+    ("MinistryofWaterandEnvironmentofYemen",
+     {"authorizedBy": ["MinistryofWaterandEnvironmentofYemen"], "compliesWith": []}),
+    (
+        "The SPICE project (Science for the protection of indonesian Coastal marine Ecosystems) was conducted and permitted under the governmental agreement between the German Federal Ministry of Education and Research (BMBF) and the Indonesian Ministry for Research and Technology (RISTEK), Indonesian Institute of Sciences (LIPI), Indonesian Ministry of Maritime Affairs and Fisheries (DKP), and Agency for the Assessment and Application of Technology (BPPT). This work was carried out in co-operation with Hassanuddin University (UNHAS, Makassar, Indonesia), Agricultural University Bogor (IPB, Bogor, Indonesia), and Jenderal Soedirman University (Purwokerto, Indonesia).",
+        {"authorizedBy": [
+            "The SPICE project (Science for the protection of indonesian Coastal marine Ecosystems) was conducted and permitted under the governmental agreement between the German Federal Ministry of Education and Research (BMBF) and the Indonesian Ministry for Research and Technology (RISTEK), Indonesian Institute of Sciences (LIPI), Indonesian Ministry of Maritime Affairs and Fisheries (DKP), and Agency for the Assessment and Application of Technology (BPPT). This work was carried out in co-operation with Hassanuddin University (UNHAS, Makassar, Indonesia), Agricultural University Bogor (IPB, Bogor, Indonesia), and Jenderal Soedirman University (Purwokerto, Indonesia)."],
+            "compliesWith": []}),
+    ("DAFF/DEA", {"authorizedBy": ["DAFF", "DEA"], "compliesWith": []}),
+    ("Ministry of Environment, Natural Resources, and Transport",
```

Comment:
This one should result in a single string "`Ministry of Environment, Natural Resources, and Transport`"

Reply:
I'm not sure how smart we can be here without regressing some of the other ones.  I'll remove the comma rule and see how many tests break and whether there is a way to tighten up that rule.